frontend/src/components/Sidebar.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import { Links } from "./nextUI/index";
+import { Container, Spacer } from "@nextui-org/react";
+
+/**
+ * Component for the sidebar
+ * @component shows a sidebar that shows users different links
+ */
+const Sidebar = () => {
+  return (
+    <aside>
+      <Container
+        display={"flex"}
+        direction={"column"}
+        css={{ padding: "3rem", minWidth: "20rem", height: "80vh" }}
+      >
+        <Links href={"/user/edit-profile"} text={"Edit Profile"} />
+        <Spacer y={0.5} />
```

Comment:
The `<Spacer>`s seem like they can be replaced with a margin on the `<Links>` or a `gap` property in the `<Container>` css